uptane-standard.md

```diff
@@ -632,7 +641,7 @@ An Uptane-compliant ECU SHALL be able to download and verify image metadata and
 
 Each ECU in a vehicle receiving over-the-air updates is either a primary or a secondary ECU. A primary ECU collects and delivers to the Director vehicle manifests ({{vehicle_version_manifest}}) containing information about which images have been installed on ECUs in the vehicle. It also verifies the time and downloads and verifies the latest metadata and images for itself and for its secondaries. A secondary ECU verifies the time and downloads and verifies the latest metadata and images for itself from its associated primary ECU. It also sends signed information about its installed images to its associated primary.
 
-All ECUs MUST verify image metadata as specified in {{metadata_verification}} before installing an image or making it available to other ECUs. A primary ECU MUST perform full verification ({{full_verification}}). A secondary ECU SHOULD perform full verification if possible. See [Uptane Deployment Considerations](#DEPLOY) for a discussion of how to choose between partial and full verification.
+All ECUs MUST verify image metadata as specified in {{metadata_verification}} before installing an image or making it available to other ECUs. A primary ECU MUST perform full verification ({{full_verification}}). A secondary ECU SHOULD perform full verification if possible. If a secondary cannot perform full verification, it SHALL perform partial verification. See [Uptane Deployment Considerations](#DEPLOY) for a discussion of how to choose between partial and full verification.
```

Comment:
This should also be at most a `SHOULD` because not all secondaries are capable of performing any verification.

Reply:
Then based on my understanding of Uptane and by definition, this ECU is not an Uptane secondary.
I think we should have further discussion on what constitutes as a Secondary if you believe this should be conditional.

Reply:
Are we on agreement on this? I think the understanding based on the last call was that we do not discuss ECUs that cannot do Uptane.